ts/src/base/Exchange.ts

```diff
@@ -1958,32 +1958,26 @@ export default class Exchange {
         }
         const market = this.markets[symbol];
         const feeSide = this.safeString (market, 'feeSide', 'quote');
-        let key = 'quote';
-        let cost = undefined;
-        const amountString = this.numberToString (amount);
-        const priceString = this.numberToString (price);
-        if (feeSide === 'quote') {
+        let useQuote = undefined;
+        if (feeSide === 'get') {
             // the fee is always in quote currency
-            cost = Precise.stringMul (amountString, priceString);
-        } else if (feeSide === 'base') {
-            // the fee is always in base currency
-            cost = amountString;
```

Comment:
here was the bug. key was not set.

Reply:
Will check it soon, thanks for fixing it! 🙂

Reply:
Thanks